apps/starter/src/ensemble/dashboard.yaml

```diff
@@ -58,6 +58,10 @@ View:
                   crossAxis: stretch
                   margin: 0 20px
                   children:
+                    - Search:
+                        placeholder: Search
+                        data: data.getData.body.users
+                        searchKey: email
```

Comment:
We will also need the ability to hook this up to API filtering and search params.